dev/start

```diff
@@ -52,8 +58,19 @@ if [[ $ENABLE_AUTHN_LDAP = true ]]; then
   docker-compose exec conjur conjurctl policy load cucumber /src/conjur-server/dev/files/authn-ldap/policy.yml
 fi
 
+if [[ $ENABLE_ROTATORS = true ]]; then
+  services="$services testdb cucumber"
+fi
+
 docker-compose up -d --no-deps $services
 
 api_key=$(docker-compose exec -T conjur conjurctl \
 	role retrieve-key cucumber:user:admin | tr -d '\r')
-docker exec -e CONJUR_AUTHN_API_KEY=$api_key $env_args -it --detach-keys 'ctrl-\' $(docker-compose ps -q conjur) bash
+
+if [[ $ENABLE_ROTATORS = true ]]; then
```

Comment:
These `docker exec`s are complicated enough that they probably deserve to be DRYed up. Maybe do something like

```
if [[ $ENABLE_ROTATORS = true ]]; then
  container_name=cucumber
else
  container_name=conjur
fi

docker exec -e CONJUR_AUTHN_API_KEY=$api_key $env_args \
    -it --detach-keys 'ctrl-\' "$(docker-compose ps -q $container_name)" bash
```

Reply:
Agree, that was me being lazy since this script wasn't part of the main code, but I'll make this change.